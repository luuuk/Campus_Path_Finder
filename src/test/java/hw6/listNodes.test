# This test tests the alphabetization functionality of the listNodes command

LoadGraph test test.tsv

ListNodes test