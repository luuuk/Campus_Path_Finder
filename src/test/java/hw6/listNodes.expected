# This test tests the alphabetization functionality of the listNodes command

loaded graph test

test contains: a b c d
